app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::SessionsController < ApplicationController
```

Comment:
why are we creating/logging in user in this controller? moreover this is api only app which wont maintain any sessions, so this is not really needed